backend/app/modules/common/timestamp_converter.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
+
+
+def timestamp_to_isoformat(timestamp_sec: int) -> str:
```

Comment:
Hopefully we will align most of the code in `legacy` to new arch soon. In this story I created a few tickets for it: https://linear.app/golemfoundation/issue/OCT-1346/octant-algorithm-overhaul. `time.py` will be affected too, so it will be possible to consolidate it then to one file

Reply:
sure, so I moved it to the mentioned time.py module, we'll be refactoring it when working with tasks